src/Vulkan/Silk.NET.Vulkan/Chain.g.cs

```diff
@@ -6903,7 +6903,7 @@ public static Chain<TChain> LoadAny<TChain>(out string errors, TChain chain)
 /// <summary>
 /// Represents a 1 element long structure chain.
 /// </summary>
-public interface IChain<TChain>
+public interface IChain<TChain> : IDisposable
```

Comment:
thinking about it more, maybe also IReadOnlyList<IChainable>? could help with some of the find-type actions on chains with unknown contents

Reply:
What would be the T for IReadOnlyList<T>? I don't think we can make this generic across chains...

Reply:
the base concrete Chain type uses IChainable

Reply:
Ah I see what you mean, yeah ok I'll do that.